apps/src/blockly/addons/cdoUtils.js

```diff
@@ -469,3 +472,21 @@ function simplifyBlockState(block, variableMap) {
 export function getBlockColor(block) {
   return block?.style?.colourPrimary;
 }
+
+export function appendSharedFunctions(startBlocksSource, functionsXml) {
```

Comment:
mind adding a comment explaining what this function does?

Reply:
Can do!